applications/crossbar/priv/api/swagger.json

```diff
@@ -11443,6 +11443,11 @@
                     "description": "tasks aging expiry in days",
                     "type": "integer"
                 },
+                "attempt_upload_output_times": {
```

Comment:
I am very wary about manually adding things to swagger.json for 2 reasons:
1. no one uses this file
2. chances are these manual edits will get deleted by the next `make apis`